spec/game_logic_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative '../lib/game_logic.rb'
+RSpec.describe Game do
```

Comment:
We need to instantiate a `Game` object in order to use it in our tests. The purpose of these tets is to make sure that our Game class works as intended. Its not really testing if we create a variable, assign it a value and then check to see if this variable is equal to this value. We want to use the Game class in our tests [`REQUIRED`].